packages/desktop-gui/src/settings/settings-spec.jsx

```diff
@@ -0,0 +1,11 @@
+/// <reference types="cypress" />
+// import Settings from './settings'
+
+// NOTE: cannot import Configuration - regeneratorRuntime undefined error
```

Comment:
This is interesting. Can you explain it? This is babel-related right?

Reply:
yeah, it is some strange async / await stuff. I need to update this example, we can actually load this